thanos/plugindefinition.yaml

```
@@ -64,9 +64,13 @@ spec:
           name: thanos.query.serviceLabels
           required: false
           type: "map"
-        - default: 
-          - prometheus-operated:10901
-          description: Standard port to be exposed by Prometheus Operator. If multiple Prometheis are deployed, this needs to be switched to prometheus' CRD name suffixed with -prometheus.
+        - default: false
+          description: Removes standard Thanos endpoints for overarching Queries 
+          name: thanos.query.standalone
+          required: false
+          type: "bool"
+        - default: [] 
+          description: "Automatically filled if Query is not standalone. If multiple Prometheis are deployed, all Prometheus services and Thanos store services need to be added to this list manually. Usually this means Prometheus'-CRD name suffixed with -prometheus and Thanos' release name suffixed with -store."
```

I would really like the required services to be added automatically in the future. 
```suggestion
          description: "Automatically filled if Query is not standalone. If multiple Prometheis are deployed, all Prometheus, Thanos Query and Thanos Store services need to be added to this list manually. e.g. kube-monitoring-prometheus:9090 or thanos-store:10901"
```

---

I suppose this was outputted by the greenhousectl tooling, but granted.

---

Well yes, we've got to think about this.